### ISSUE TITLE
Allow setting Hasura actions bind address

### ISSUE THREAD

**Issue by makeworld-the-better-one** [forbole/bdjuno#449]
## Problem description 

Right now the only configuration option for hasura actions with bdjuno is the port:

```yaml
actions:
    port: 3000
```

This means that on my server the Hasura actions port is publicly exposed. I'm not sure if this is a security issue or not (is it?), but in any case ports should be closed unless needed.

## Implementation proposal

A bind address option could be added, so that the hasura actions service was only available on localhost, for example:
```yaml
actions:
    port: 3000
    addr: "127.0.0.1"
```

**Comment by RiccardoM** [comment]
I honestly have a hard time understanding what's the problem here. Hasura actions are started listening to the url `:<port>`. Can't you simply avoid having the specified `<port>` exposed publicly using a firewall or something similar?

**Comment by makeworld-the-better-one** [comment]
Yes, a firewall can be used. But it's pretty standard to have a host option as well, and makes it easy to protect the service. Just my 2 cents.

**Comment by RiccardoM** [comment]
@makeworld-the-better-one Ok, I see your point. I will move this to [Juno](https://github.com/forbole/juno) since it might be even more useful there. Do you think you can open a PR for this?
